package.json

```diff
@@ -1,6 +1,6 @@
 {
   "id" : "netuitive.packages.java",
-  "version" : "1.7.0",
+  "version" : "1.9.0",
```

Comment:
Could you bump this directly on `develop` once you merge this PR?

Reply:
✔️